internal/controller/backupconfig_controller.go

```diff
@@ -18,14 +18,19 @@ package controller
 
 import (
 	"context"
+	"fmt"
+	"time"
 
+	cnpgv1 "github.com/cloudnative-pg/cloudnative-pg/api/v1"
 	"github.com/go-logr/logr"
 	v1beta1 "github.com/wal-g/cnpg-plugin-wal-g/api/v1beta1"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const backupConfigFinalizerName = "cnpg-plugin-wal-g.yandex.cloud/backup-config-cleanup"
```

Comment:
It is better to declare it to `api/v1beta1/backupconfig_types.go` similarly to `Backup` finalizer name declared in `api/v1beta1/backup_types.go`.

This will make finalizer name reusable for other projects which may want to reference it